webapp/channels/src/components/user_settings/modal/user_settings_modal.tsx

```diff
@@ -64,7 +65,7 @@ class UserSettingsModal extends React.PureComponent<Props, State> {
         super(props);
 
         this.state = {
-            active_tab: props.isContentProductSettings ? 'notifications' : 'profile',
+            active_tab: props.activeTab ?? (props.isContentProductSettings ? 'notifications' : 'profile'),
```

Comment:
0/5, but for readability sake I would make this assignment a bit more verbose, but trying to make an example, I didn't like it either. If you find an elegant way for this to be more readable, please do. If not, feel free to ignore the comment.

Reply:
@larkox Honestly, with this new functionality, we may as well consider eliminating `isContentProductSettings` altogether and letting the component's user pass the expected tab identifier as prop. I think that's the proper simplification but I was a little concerned about touching more webapp stuff.

Reply:
Fair enough :+1: